// css-to-react-native/src/transforms/index.js
import {
  IDENT,
  WORD,
  COLOR,
  LENGTH,
  UNSUPPORTED_LENGTH_UNIT,
  PERCENT,
  AUTO,
} from 'css-to-react-native/src/tokenTypes'
import border from 'css-to-react-native/src/transforms/border'
import boxShadow from 'css-to-react-native/src/transforms/boxShadow'
import flex from 'css-to-react-native/src/transforms/flex'
import flexFlow from 'css-to-react-native/src/transforms/flexFlow'
import font from 'css-to-react-native/src/transforms/font'
import fontFamily from 'css-to-react-native/src/transforms/fontFamily'
import placeContent from 'css-to-react-native/src/transforms/placeContent'
import textDecoration from 'css-to-react-native/src/transforms/textDecoration'
import textDecorationLine from 'css-to-react-native/src/transforms/textDecorationLine'
import textShadow from 'css-to-react-native/src/transforms/textShadow'
import transform from 'css-to-react-native/src/transforms/transform'
import { directionFactory, parseShadowOffset } from 'css-to-react-native/src/transforms/util'

const background = tokenStream => ({
  backgroundColor: tokenStream.expect(COLOR),
})
const borderColor = directionFactory({
  types: [COLOR],
  prefix: 'border',
  suffix: 'Color',
})
const borderRadius = directionFactory({
  directions: ['TopLeft', 'TopRight', 'BottomRight', 'BottomLeft'],
  prefix: 'border',
  suffix: 'Radius',
})
const borderWidth = directionFactory({ prefix: 'border', suffix: 'Width' })
const margin = directionFactory({
  types: [LENGTH, UNSUPPORTED_LENGTH_UNIT, PERCENT, AUTO],
  prefix: 'margin',
})
const padding = directionFactory({ prefix: 'padding' })
const fontVariant = tokenStream => ({
  fontVariant: [tokenStream.expect(IDENT)],
})
const fontWeight = tokenStream => ({
  fontWeight: tokenStream.expect(WORD), // Also match numbers as strings
})
const shadowOffset = tokenStream => ({
  shadowOffset: parseShadowOffset(tokenStream),
})
const textShadowOffset = tokenStream => ({
  textShadowOffset: parseShadowOffset(tokenStream),
})

export default {
  background,
  border,
  borderColor,
  borderRadius,
  borderWidth,
  boxShadow,
  flex,
  flexFlow,
  font,
  fontFamily,
  fontVariant,
  fontWeight,
  margin,
  padding,
  placeContent,
  shadowOffset,
  textShadow,
  textShadowOffset,
  textDecoration,
  textDecorationLine,
  transform,
}
